commands/bootstrap/index.js

```diff
@@ -36,6 +36,8 @@ class BootstrapCommand extends Command {
 
   initialize() {
     const { registry, rejectCycles, npmClient = "npm", npmClientArgs, mutex, hoist } = this.options;
+    const filteredLength = this.filteredPackages.length;
+    this.packageCountLabel = `${filteredLength} package${filteredLength > 1 's' : ''}`;
```

Comment:
Er, you missed a `?` in the ternary.

Reply:
@evocateur Sorry for that. Fixed in the latest commit.